skeletons/web-extension/content-script.js

```diff
@@ -24,6 +24,7 @@
    *     https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API
    */
   window.addEventListener('message', function(event) {
+    console.log('message', event);
```

Comment:
Do we want to keep all of these console logs or were they for testing?

Reply:
ready